itests/util/src/main/java/org/apache/hadoop/hive/ql/hooks/VerifyOutputTableLocationSchemeIsFileHook.java

```diff
@@ -24,10 +24,10 @@ public class VerifyOutputTableLocationSchemeIsFileHook implements ExecuteWithHoo
 
   @Override
   public void run(HookContext hookContext) {
-    for (WriteEntity output : hookContext.getOutputs()) {
-      if (output.getType() == WriteEntity.Type.TABLE) {
-        String scheme = output.getTable().getDataLocation().toUri().getScheme();
-        Assert.assertTrue(output.getTable().getTableName() + " has a location which has a " +
+    for (ReadEntity input : hookContext.getInputs()) {
```

Comment:
the logic here seem to have changed ; and its not aligned anymore with the name of the class

is this change intended? maybe you wanted to add a new hook which check for the input?

Reply:
Yeah, this change is an intended one. This class is only used in rename_table_location.q file and it is used  to verify the table object and its location. Previously, output and input had the old table objects for authorization. Now, we would like to have the old table in the input list and the new database and table object in the output list.